cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ClusterLoadParameters.java

```diff
@@ -13,13 +13,20 @@
 /**
  * Parameters for {@link com.linkedin.kafka.cruisecontrol.servlet.EndPoint#LOAD}
  *
+ * <ul>
+ *   <li>Note that parameter "time" is mutually exclusive to parameter "start" and "end" -- i.e. they cannot be used together.</li>
```

Comment:
Nit: If we make the changes based on the prev. comment on use of start and time, maybe we can update this JavaDoc.